.node-version

```diff
@@ -1 +1 @@
-10.24.1
+16.13.2
```

Comment:
Actually we prefer to keep the package compatible with node 10 as long as there is no express need to bump to a newer version. Could you revert this change?